en/gps_compass/README.md

```diff
@@ -23,34 +23,35 @@ On small vehicles an external compass is almost always required.
 PX4 should work with any unit that communicates via the u-blox, MTK Ashtech or Emlid protocols, or via UAVCAN.
 This list contains GNSS/Compass units that have been tested by the PX4 dev team, or which are popular within the PX4 community.
 
-Device | GPS | Compass | [RTK](../gps_compass/rtk_gps.md) | [GPS Yaw Output](#configuring-gps-as-yaw-heading-source) | [Dual F9P GPS Heading](../gps_compass/u-blox_f9p_heading.md)
+Device | GPS | Compass | [RTK](../gps_compass/rtk_gps.md) | [GPS Yaw Output](#configuring-gps-as-yaw-heading-source) | [Dual F9P GPS Heading](../gps_compass/u-blox_f9p_heading.md) | PPK(Post-Processing Kinematic) 
```

Comment:
The addition of "PPK (Post-Processing Kinematic) " appears to have broken the table. Is it worth it given that you have not marked any devices as having this feature? 

What is the feature? When do you expect it? Does it have any additional setup associated?